cmd/influxdb/main.go

```diff
@@ -12,8 +12,8 @@ import (
 	"github.com/mainflux/mainflux"
 	log "github.com/mainflux/mainflux/logger"
 	"github.com/mainflux/mainflux/writers"
-	influxdb "github.com/mainflux/mainflux/writers/influxdb"
-	nats "github.com/nats-io/go-nats"
+	"github.com/mainflux/mainflux/writers/influxdb"
+	"github.com/nats-io/go-nats"
```

Comment:
Please return `nats` alias. I think that improves readability (You don't need to open files from `github.com/nats-io/go-nats` to see package name).